src/functionalTest/resources/features/F-013/S-216.td.json

```diff
@@ -0,0 +1,93 @@
+  "users": {
+    "invokingUser": {
+      "_extends_": "AmBeftaUser2"
+    }
+  },
+  "request": {
+    "headers": {
+      "ServiceAuthorization": "${[scenarioContext][customValues][generateS2STokenForXui]}"
+    },
+    "body": {
+      "roleRequest": {
+        "assignerId": "${[scenarioContext][testData][users][invokingUser][id]}",
+        "reference": "1662666441316010/challenged-access-judiciary/${[scenarioContext][testData][users][invokingUser][id]}",
+        "process" : "challenged-access",
+        "replaceExisting" : true
+      },
+      "requestedRoles": [
+        {
+          "_extends_": "RequestedRole",
+          "roleType" : "CASE",
+          "grantType" : "CHALLENGED",
+          "classification" : "PUBLIC",
+          "attributes": {
+            "jurisdiction": "CIVIL",
+            "caseId": "1662666441316010"
+          },
+          "roleName": "challenged-access-judiciary",
+          "roleCategory": "JUDICIAL",
+          "actorId": "${[scenarioContext][testData][users][invokingUser][id]}",
+          "beginTime": "${[scenarioContext][customValues][today]}T00:00:00Z",
+          "endTime": "${[scenarioContext][customValues][tomorrow]}T00:00:00Z",
+          "readOnly": true,
+          "notes" : [{
+            "userId": "${[scenarioContext][testData][users][invokingUser][id]}",
+            "time": "2022-01-01T00:00Z",
+            "comment": "Need Access to case number 1662666441316010 for a month"
+          }]
+        }
+      ]
+    }
+  },
+  "expectedResponse": {
+    "_extends_": "Common_201_Response",
+    "headers": {
+      "Content-Length": "[[ANYTHING_PRESENT]]",
+      "Content-Type": "application/json"
+    },
+    "body": {
+      "roleAssignmentResponse": {
+        "roleRequest": {
+          "_extends_": "ResponseRoleRequest",
+          "reference": "${[scenarioContext][testData][request][body][roleRequest][reference]}",
+          "process" : "${[scenarioContext][testData][request][body][roleRequest][process]}",
+          "replaceExisting": true
+        },
+        "requestedRoles": [
+          {
+            "_extends_": "ResponseRequestedRole",
+            "process" : "challenged-access",
+            "actorId": "${[scenarioContext][testData][request][body][requestedRoles][0][actorId]}",
+            "roleName": "${[scenarioContext][testData][request][body][requestedRoles][0][roleName]}",
+            "roleCategory": "${[scenarioContext][testData][request][body][requestedRoles][0][roleCategory]}",
+            "roleType": "${[scenarioContext][testData][request][body][requestedRoles][0][roleType]}",
+            "grantType": "${[scenarioContext][testData][request][body][requestedRoles][0][grantType]}",
+            "classification" : "${[scenarioContext][testData][request][body][requestedRoles][0][classification]}",
+            "readOnly": "${[scenarioContext][testData][request][body][requestedRoles][0][readOnly]}",
+            "attributes": {
+              "caseId": "1662666441316010",
```

Comment:
For Challenged Access, we need to validate region id from attribute.
The rule is Requestor ORG role region id is not equal to case region id.
In this test, we didn't provide the region id.
Please add the region id in your test and validate.